launcher/src/se/llbit/chunky/launcher/JavaFxLocator.java

```diff
@@ -0,0 +1,82 @@
+package se.llbit.chunky.launcher;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.management.ManagementFactory;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class JavaFxLocator {
+
+
+  private static final String[] javafxPathCandidates;
+
+  static {
+    String[] javafxPathCandidates1;
+    try {
+      javafxPathCandidates1 = new String[]{
+              "C:/Program Files/openjfx/lib",
+              System.getProperty("java.home", "") + File.separator + "lib", // java home
+              new File(ChunkyLauncher.class.getProtectionDomain().getCodeSource().getLocation().toURI())
+                      .toPath().getParent().toAbsolutePath().toString() + File.separator + "lib", // directory of the .jar
+              System.getProperty("user.dir", "") + File.separator + "lib", // working directory
+              "/usr/share/openjfx/lib",
+              "/usr/lib/jvm/java-11-openjdk/lib",
+              "/usr/lib/jvm/java-12-openjdk/lib",
+              "/usr/lib/jvm/java-13-openjdk/lib",
+              "/usr/lib/jvm/java-14-openjdk/lib",
+              "/usr/lib/jvm/java-15-openjdk/lib",
+      };
+    } catch(URISyntaxException e) {
+      e.printStackTrace();
+      javafxPathCandidates1 = new String[0];
```

Comment:
The try…catch could be moved around `new File(ChunkyLauncher.class.getProtectionDomain().getCodeSource().getLocation().toURI())
                      .toPath().getParent().toAbsolutePath().toString() + File.separator + "lib", ` here to still use the other candidates. This isn't some hot code path, so you could probably use a `List<String>` to add the candidates dynamically.